.github/actions/pre-init/action.yaml

```diff
@@ -40,12 +40,11 @@ runs:
         echo 'AWS_ACCESS_KEY_ID=${{ inputs.sccache_aws_access_key_id }}' >> $GITHUB_ENV
 
     # Cache generated Bolt files in order to prevent needless rebuilding
-    - name: Bolt Cache
-      uses: actions/cache@v3
-      with:
-        key: ${{ runner.os }}-bolt-gen
-        path: |
-          svc/pkg/region/ops/config-get/gen
+    # - name: Bolt Cache
+    #   uses: actions/cache@v3
+    #   with:
+    #     key: ${{ runner.os }}-bolt-gen
+    #     path:
```

Comment:
If this is getting removed for now and maybe added back later, can we create a ticket for future work to fix this?